platform.md

```diff
@@ -1062,15 +1049,30 @@ For more information on build reproducibility see [https://reproducible-builds.o
 
 [run-image]
   reference = "<image reference>"
+  [target]
+  id = "<target identifer>"
+  os = "<OS name>"
+  arch = "<architecture>"
+  variant = "<architecture variant>"
+  [target.distribution]
+  name = "<OS distribution name>"
+  version = "<OS distribution version>"
 
 [build-image]
   reference = "<image reference>"
 ```
 
 Where:
-- `previous-image.reference` MUST be either a digest reference to an image in an OCI registry or the ID of an image in a docker daemon
+- `image.reference` MUST be either a digest reference to an image in an OCI registry or the ID of an image in a docker daemon
+- `image.metadata` MUST be the TOML representation of the layer [metadata label](#iobuildpackslifecyclemetadata-json)
 - `run-image.reference` MUST be either a digest reference to an image in an OCI registry or the ID of an image in a docker daemon
-- `previous-image.metadata` MUST be the TOML representation of the layer [metadata label](#iobuildpackslifecyclemetadata-json)
+- `run-image.target.id` is optional and MUST be the value of the label `io.buildpacks.id`
+- `run-image.target` contains the [target data](#target-data) for the image
+  - If target data is missing but the image contains the label `io.buildpacks.stack.id` with value `io.buildpacks.stacks.bionic`, the lifecycle SHALL assume the following values:
+    - `run-image.target.os = "linux"`
+    - `run-image.target.arch = "x86_64"`
```

Comment:
Do we want to use values here that match `go tool dist list`?

Reply:
```suggestion
    - `run-image.target.arch = "arm64"`
```
yes. I think that's derived from [$GOARCH](https://go.dev/doc/install/source#environment). Per the [RFC](https://github.com/buildpacks/rfcs/blob/main/text/0096-remove-stacks-mixins.md), we should align with [OCI Image Config Spec](https://github.com/opencontainers/image-spec/blob/main/config.md):

> architecture string, REQUIRED
>
> The CPU architecture which the binaries in this image are built to run on. Configurations SHOULD use, and implementations SHOULD understand, values listed in the Go Language document for [GOARCH](https://golang.org/doc/install/source#environment).